README.md

```diff
@@ -1,3 +1,5 @@
+[![nbirillo](https://circleci.com/gh/nbirillo/ast-transformations/tree/master.svg?style=shield)](https://app.circleci.com/pipelines/github/nbirillo/ast-transformations?branch=master)
```

Comment:
Cannot add a comment to the line 9, but let's say "Just clone the repo by running `git clone https://github.com/nbirillo/ast-transformations.git`"